content/applications/inventory_and_mrp/repairs/repair_orders.rst

```diff
@@ -0,0 +1,258 @@
+      :alt: Delivered and Quantity columns on sales order after return.
+
+.. _repairs/repair_orders/repair:
+
+Create repair order
+===================
+
+Once products have been returned, their repairs can be tracked by creating a repair order (RO).
+
+To create a new |RO|, navigate to :menuselection:`Repairs app`, and click :guilabel:`New`. This
+opens a blank |RO| form.
+
+.. image:: repair_orders/repair-orders-left-hand-form.png
+   :align: center
+   :alt: Left-hand side of blank repair order form.
+
+On this form, begin by selecting a :guilabel:`Customer`. The customer selected should be for whom
+the order will be invoiced and delivered.
+
+In the :guilabel:`Product to Repair` field, click the drop-down menu to select the product that
+needs repair. If necessary, click :guilabel:`Search More...` to open a :guilabel:`Search: Product to
+Repair` pop-up window, and browse all products in the database.
+
+Once a :guilabel:`Product to Repair` is selected, a new :guilabel:`Product Quantity` field appears
+below it. In that field, enter the quantity (in a `0.00` format) of the product that requires
+repair.
+
+To the right of that value, click the drop-down list to select the unit of measure (UoM) for the
+product.
+
+In the :guilabel:`Return` field, click the drop-down menu and select the return order from which the
+product to be repaired comes from.
+
+Tick the :guilabel:`Under Warranty` checkbox, if the product being repaired is covered by a
+warranty. If ticked, the :guilabel:`Customer` is not charged for all the parts used in the repair
+order.
+
+In the :guilabel:`Scheduled Date` field, click the date to reveal a calendar popover window. From
+this calendar, select a date for the repair, and click :guilabel:`Apply`.
+
+.. image:: repair_orders/repair-orders-right-hand-form.png
+   :align: center
+   :alt: Right-hand side of blank repair order form.
+
+In the :guilabel:`Responsible` field, click the drop-down menu and select the user who should be
+responsible for the repair.
+
+In the :guilabel:`Company` field, if in a multi-company environment, select which company this |RO|
+belongs to.
+
+In the :guilabel:`Tags` field, click the drop-down menu and select which tags should be applied to
+this |RO|.
+
+Parts tab
+---------
+
+Add, remove, or recycle parts in the :guilabel:`Parts` tab. To do so, click :guilabel:`Add a line`
+at the bottom of the form.
+
+In the :guilabel:`Type` column, click the box to reveal three options to choose from:
+:guilabel:`Add` (selected by default), :guilabel:`Remove`, and :guilabel:`Recycle`.
+
+.. image:: repair_orders/repair-orders-type-column.png
+   :align: center
+   :alt: Type column options or new part under Parts tab.
+
+Choosing :guilabel:`Add` adds this part to the |RO|.
+
+Choosing :guilabel:`Remove` removes this part from the |RO|.
+
+Choosing :guilabel:`Recycle` recycles this part from the |RO|, designating it for later use or to be
+repurposed for another use in the warehouse.
```

Comment:
It's clear to me that this is what these options are for— the bigger question I have is what exactly are we doing on the _Parts_ tab? What's it for? For what purpose would we Add, Remove, or Recycle parts here?